### ISSUE TITLE
Implement quo2 - Wallet - Slider button component

### ISSUE THREAD

**Issue by cammellos** [status-im/status-react#16207]
Implement the wallet slider button as per designs:

https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=19085-205591&t=F0kGMzHP0FA98MTb-0

Please place the implementation in the quo2 preview.
Have a word/chat with our designers before starting to implement to clarify the interactions.

The animation is not specified by the design and that's where we would like to see a good UX.

Blur view is out of scope.
